R/AllGenerics.R

```diff
@@ -147,3 +147,6 @@ setGeneric("splitByFile", function(object, f, ...) standardGeneric("splitByFile"
 setGeneric("productMz", function(object, ...) standardGeneric("productMz"))
 ## setGeneric("aggregationFun", function(object, ...)
 ##     standardGeneric("aggregationFun"))
+
+setGeneric("extractMsData", function(object, ...)
```

Comment:
I'm using here a method instead of a function, because in `xcms` I have a `extractMsData,XCMSnExp` method that has some additional functionality (ensures that adjusted retention times are used if present).